utils/termdb/buildTermdb.js

```diff
@@ -245,10 +245,7 @@ function loadTermsFile(scriptArg) {
 	 */
 	const terms = []
 	const parent_ids = new Set() //To store parent_id for each term in terms
-	for (const line of fs
```

Comment:
please delete this file and use sjpp/utils/termdb/buildTermdb.js instead. this one was accidentally left here before repo switch. sjpp version has far more features

Reply:
There is only this pp file Xin, in sjpp a bundle version is generated, but I could not find the duplicate you mentioned.

Reply:
Then I am trying to move it to sjpp/utils/termdb and compile it to test the changes done but I need to update the import paths. I am having compilation errors.

Reply:
What I see now is this error `	initBinConfig is not a function` when running `node ~/dev/sjpp/utils/termdb/buildTermdb.bundle.js terms=terms.json annotation3Col=annotations.txt dbfile=db
sampleAncestry=sample_ancestry.tsv`

Reply:
@airenzp Could you console.log `initBinConfig`? 

Also, may be unrelated but good to update since it's needed anyway: in `sjpp/utils/termdb/webpack.config.js` line 33, set the `targets.node: 20` instead of `12`.

![Screenshot 2024-03-19 at 9 20 43 PM](https://github.com/stjude/proteinpaint/assets/411031/09d1aa39-6584-422c-af99-c5650adf50e3)

Reply:
I could fix the initBinConfig error, and updated now the node version, thanks.